examples/blender/blender.py

```diff
@@ -102,7 +110,7 @@ async def worker(ctx: WorkContext, tasks):
                 )
 
     # Iterator over the frame indices that we want to render
-    frames: range = range(0, 60, 10)
+    frames: range = range(0, 60000, 10)
```

Comment:
again, I'd add parameters that vary these values -> `--start-frame`, `--end-frame`, `--frame-step` ? ...